app/views/coronavirus_landing_page/components/shared/_country_guidance.html.erb

```diff
@@ -0,0 +1,17 @@
+<% guidance ||= nil %>
+
+<div
+  class="govuk-body"
+  data-module="gem-track-click"
+  data-track-category="pageElementInteraction"
+  data-track-action="AccordionClick"
+  data-track-links-only>
+
+  <% if guidance.present? && guidance["links"].present? %>
+      <% guidance["links"].map do |link| %>
+        <p class="govuk-body">
+          <%= link_to(link["label"], link["url"], class: "govuk-link") %>
+        </p>
```

Comment:
This should probably now be a list, rather than a series of paragraphs. You can use the list from the [Design System here](https://design-system.service.gov.uk/styles/typography/#lists), something like:

```
<ul class="govuk-list">
  <% guidance["links"].map do |link| %>
    <li><%= link_to(link["label"], link["url"], class: "govuk-link") %></li>
  <% end %>
</ul>
```

Reply:
@andysellick Thanks for the suggestion.

We (@gclssvglx and I) added a margin-top to the "ul" and that works too!

```ruby
  <% if guidance.present? && guidance["links"].present? %>
      <ul class="govuk-list govuk-!-margin-top-3">
      <% guidance["links"].map do |link| %>
        <li>
          <%= link_to(link["label"], link["url"], class: "govuk-link") %>
        </li>
      <% end %>
      </ul>
  <% end %>
```

Reply:
👍 I've added the margin_bottom setting to heading and fixed up.